connectors/sns/element-templates/aws-sns-inbound-start-event.json

```diff
@@ -0,0 +1,141 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "SNS HTTPS Subscription",
+  "id": "io.camunda.connectors.inbound.AWSSNS.StartEvent.v1",
+  "description": "Receive events from AWS SNS",
+  "version": 1,
+  "documentationRef": "https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/aws-sns/",
+  "category": {
+    "id": "connectors",
+    "name": "Connectors"
+  },
+  "appliesTo": [
+    "bpmn:StartEvent"
+  ],
+  "elementType": {
+    "value": "bpmn:StartEvent"
+  },
+  "groups": [
+    {
+      "id": "subscription",
+      "label": "Subscription Configuration"
+    },
+    {
+      "id": "activation",
+      "label": "Activation"
+    },
+    {
+      "id": "variable-mapping",
+      "label": "Variable Mapping"
+    }
+  ],
+  "properties": [
+    {
+      "type":"Hidden",
+      "value":"io.camunda:connector-sns-inbound:1",
```

Comment:
Lets use `io.camunda:aws-sns-inbound:1` as we do for the other AWS connectors.

Reply:
Done